src/scripts/generateToc.js

```diff
@@ -1,10 +1,25 @@
 const fs = require('fs');
```

Comment:
Will this script end up being part of `yarn generate`?

Note that `yarn generate` now moves much of the docs data to `generated/` -- that may be a "head start" on getting it imported into the docs site.

Reply:
Yes, I believe it should.

> Note that `yarn generate` now moves much of the docs data to `generated/`

That's awesome. Thanks :)